### ISSUE TITLE
Mb Web client - Release date metadata error

### ISSUE THREAD

**Issue by nodiaque** [MediaBrowser/Emby#398]
Hello,

When I fetch data or enter the release date of a movie manualy, the date saved is 1 day off. Let’s say I input 2002-08-24, once saved, if I go back to the movie and edit the metadata again, it will be 2002-08-23. Funny thing, it will happen on every save. Thus, each time you edit then save, the date is lowered.

Thanks

Edit: The xml file is OK, but the date showed by the MB web client is 1 day off. Thus, the date will start lowering after the second edit.

In short,

You enter release date 2002-08-24. You click save, the xml contain 2002-08-24. You go back to the movie and click edit again. The Web Client show 2002-08-23, but the xml file have 2002-08-24. Because of that, when you save, the date in the XML file will become 2002-08-23.
